schema-guide.md

```diff
@@ -531,18 +577,19 @@ TODO explain why we use definitions
     country: DE
     ```
 
-### `definitions.date`
+### *`definitions.date`*
 
-`definitions` and its subkeys should not be used directly in `CITATION.cff` files.
-
-- **type**: `string`
+- **type**: Nonempty `string`
 - **required**: `false`
-- **description**: A date. Format is 4-digit year, 2-digit month, 2-digit day of month, separated by dashes.
+- **description**: A date. Format is 4-digit year, 2-digit month, 2-digit day of month, separated by dashes, and double-quoted or unquoted.
 - **usage**:<br><br>
     ```yaml
     date-released: "2020-01-31"
     ```
     ```yaml
+    date-released: 2020-01-31
```

Comment:
This one is a bit risky, because it depends on how exactly downstream tooling has followed the instruction of dealing with dates. I prefer to not have this example, to help people write dates as strings

Reply:
Okay.

Reply:
Done in 04351bdab600fdd0f70eb9e1ac1317fea7417baf